api/proto/dsse.proto

```diff
@@ -16,16 +16,26 @@ syntax = "proto3";
 package dev.sigstore.rekor.v2;
 
 import "google/api/field_behavior.proto";
+import "sigstore_common.proto";
+import "envelope.proto";
 
 import "verifier.proto";
 
 option go_package = "github.com/sigstore/rekor-tiles/pkg/generated/protobuf";
 
 // A request to add a dsse entry to the log
-message DSSERequest {
-    // A DSSE envelope (as defined by https://github.com/secure-systems-lab/dsse/blob/master/envelope.proto)
-    // in stringified JSON
-    string envelope = 1 [(google.api.field_behavior) = REQUIRED];
-    // collection of verification material used to verify
-    repeated Verifier verifier = 2;
+message DSSERequestV0_0_2 {
+    // A DSSE envelope
+    io.intoto.Envelope envelope = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
[This envelope](https://github.com/sigstore/protobuf-specs/blob/7e4a1adcf03949f345a9f37c62de8a8df8dac128/protos/envelope.proto#L41) includes signatures, so would you want only the payload and payload type?

Reply:
LGTM, at the time I was concerned that we would duplicate signatures with a `SignatureAndVerifier` in the request